wrapping/test/CMakeLists.txt

```diff
@@ -39,3 +39,36 @@ itk_python_add_test(NAME PythonSpectra1DNormalizeImageFilterTest
 itk_python_add_test(NAME PythonCurvilinearArraySpecialCoordinatesImageTemplateMethodsTest
   COMMAND PythonCurvilinearArraySpecialCoordinatesImageTemplateMethodsTest.py
   )
+
+itk_python_add_test(NAME PythonBackscatterImageFilterTest0
+  TEST_DRIVER_ARGS
+  --compare
+    DATA{${test_baseline_dir}/itkBackscatterImageFilterTest0Output.mha}
+    ${ITK_TEST_OUTPUT_DIR}/PythonBackscatterImageFilterTest0.mha
+  COMMAND PythonBackscatterImageFilterTest.py
```

Comment:
`PythonBackscatterImageFilterTest.py` is referenced but doesn't seem to be present, was it added to Git history?